test/pytest/test_xpedite/test_profiler/generateBaseline.py

```diff
@@ -80,7 +80,7 @@ def replaceWorkspace(filePath, workspace, destination):
   with open(destination, 'w') as fileHandle:
     fileHandle.write(appInfoStr)
 
-def generateBaseline(context, scenario, runDir):
+def generateBaseline(context, scenarios, runDir):
```

Comment:
I changed the name to "scenarios" because we have that in other places, just one of them needs to have the name changed (either the parameter or the variable) because of this
`R:103,19: Redefining argument with the local name 'scenario' (redefined-argument-from-local)`

Reply:
you don't need to redefine it. you could just use _

` with scenarios as _:`